.github/workflows/build.yml

```diff
@@ -48,6 +48,54 @@ jobs:
           CHANGED="$(EXTRA=build .github/files/list-changed-projects.sh)"
           echo "::set-output name=projects::${CHANGED}"
 
+      - name: Check if a WordPress.com test reminder comment exists
+        id: check-test-reminder-comment
+        uses: actions/github-script@v6
+        if: ${{ github.event_name == 'pull_request' }}
+        with:
+          result-encoding: string
+          script: |
+            const query = `query($owner:String!, $name:String!, $number:Int!) {
+              repository(owner:$owner, name:$name){
+                pullRequest(number:$number) {
+                  comments(first: 100) {
+                    nodes {
+                      body
+                    }
+                  }
+                }
+              }
+            }`;
+            const variables = {
+              owner: context.repo.owner,
+              name: context.repo.repo,
+              number: context.issue.number,
+            };
+            const comments = await github.graphql(query, variables);
+            let hasComment = false;
+            comments.repository.pullRequest.comments.nodes.map( comment => {
+              if ( comment.body.includes( '<!-- wpcom-reminder-comment -->' ) ) {
+                hasComment = true;
+              }
+            } );
+
+            return hasComment;
```

Comment:
Could simplify this part, I think
```suggestion
            return comments.repository.pullRequest.comments.nodes.some( comment => comment.body.includes( '<!-- wpcom-reminder-comment -->' ) );
```

Reply:
👍 done in b2e789005a734e8ee57de9d5dfc954c97b89f689